source/_integrations/zwave_js.markdown

```diff
@@ -20,56 +20,66 @@ ha_codeowners:
 ha_domain: zwave_js
 ---
 
-This integration allows you to control a Z-Wave network via [Z-Wave JS](https://zwave-js.github.io/node-zwave-js/#/).
+This integration allows you to control a Z-Wave network via the [Z-Wave JS](https://zwave-js.github.io/node-zwave-js/#/) driver and it's the successor of the [previous Z-Wave integration](https://www.home-assistant.io/integrations/zwave/) in Home Assistant, which is now considered deprecated.
 
 ## Requirements
```

Comment:
Above this, I would just have a getting started. Something like this, glued to the first sentence:

---

To add Z-Wave JS to your installation, plug the Z-Wave stick into the device that runs Home Assistant. Then Go to **Configuration >> Integrations** in the UI. Click the "add integration" button in the bottom right and from the list of integrations, select "Z-Wave JS" and follow the instructions shown. 

Note: A new network key is automatically generated for you. If this Z-Wave stick has already paired with secure devices, you need to enter the previously used network key.

## Advanced installation instructions

The above instructions won't work if you are using Home Assistant Container, Home Assistant Core, or you don't want to use the built-in Z-Wave JS Server add-on.

…

Reply:
The text is adjusted. First section is the quick start now.